packages-ts/starknet-gauntlet-emergency-protocol/test/utils/index.ts

```diff
@@ -0,0 +1,29 @@
+import { CompiledContract, json } from 'starknet'
+import fs from 'fs'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+
+export const loadContract = (name: string): CompiledContract => {
```

Comment:
Not sure what this whole file is about. Are we using it somewhere? And if yes, the `loadContract` duplicate is unnecessary.